### DISCUSSION TITLE
sio.emit is unable to emit messages to the frontend

### DISCUSSION THREAD

**Discussion by zjr10101** [miguelgrinberg/python-socketio#1387] [Q&A]
I created the server with socketio.Server(), but after I called the function method in the while loop or if test, I used the sio.emit() method to send a message to the client, but the client could not receive the message from the server. I am sure the sio.emit() code passed. Even with a breakpoint in front of sio.emit(), the client still won't receive the message.
I'm not sure if the server-side sio.emit didn't send the message or if the client didn't receive the message.

**Comment by miguelgrinberg** [comment]
It's really impossible for me to know what's wrong from a high-level description. I recommend that you start from one of the examples in this repository and once you have it working modify it according to your needs.

**Reply by zjr10101** [reply]
Ok, thanks for the reply.
